src/System.ServiceProcess.ServiceController/tests/SafeServiceControllerTests.cs

```diff
@@ -10,6 +10,7 @@ public static class SafeServiceControllerTests
     {
         private const string KeyIsoSvcName = "KEYISO";
 
+        [ActiveIssue(19223)]
```

Comment:
This one isn't specific to Nano, but I think it's fine to port.

Reply:
Yes please